test/gtest/kernel_tuning_net.cpp

```diff
@@ -0,0 +1,99 @@
+#include <miopen/miopen.h>
+#if MIOPEN_ENABLE_AI_KERNEL_TUNING
+#include "../tensor_holder.hpp"
+#include <gtest/gtest.h>
+#include <iostream>
+#include <miopen/convolution.hpp>
+#include <miopen/problem_description.hpp>
+#include <miopen/solver.hpp>
+#include "get_handle.hpp"
+#include <unordered_map>
+
+static std::vector<std::vector<std::vector<int>>> gfx908_ConvAsm1x1U_tensor_shapes{
+    {{256, 2048, 7, 7}, {512, 2048, 1, 1}, {256, 512, 7, 7}},
+    {{512, 192, 56, 56}, {288, 192, 1, 1}, {512, 288, 56, 56}},
+    {{1, 4, 2, 2}, {4, 4, 1, 1}, {1, 4, 2, 2}}};
+
+static std::vector<miopenDataType_t> gfx908_ConvAsm1x1U_data_types{
+    miopenHalf, miopenFloat, miopenFloat};
+
+static std::vector<miopen::conv::Direction> gfx908_ConvAsm1x1U_directions{
+    miopen::conv::Direction::Forward,
+    miopen::conv::Direction::BackwardData,
+    miopen::conv::Direction::Forward};
+
+static std::vector<std::string> gfx908_ConvAsm1x1U_expected_configs{
+    "2,8,4,16,1,4,1,4", "1,16,1,64,2,2,1,4", ""};
+
+static std::vector<bool> gfx908_ConvAsm1x1U_expected_valid{true, true, false};
```

Comment:
Note for general uses:
```suggestion
static std::array<bool, 3> gfx908_ConvAsm1x1U_expected_valid{true, true, false};
```
Although wouldn't matter much in a test. Same for the rest of small const size vectors.